src/constants.ts

```diff
@@ -93,3 +96,112 @@ export const CUM_EXP_TABLE = EXP_TABLE.map(
             (sum += value)
     )(0)
 );
+
+export const VOTE_BONUS_DURATION = 1;
```

Comment:
Should we move related values into namespaces here?

Reply:
Eventually, this can come later though